mithril-client/src/client.rs

```diff
@@ -20,8 +22,13 @@ use crate::snapshot_client::SnapshotClient;
 use crate::snapshot_downloader::{HttpSnapshotDownloader, SnapshotDownloader};
 use crate::MithrilResult;
 
+#[cfg(target_family = "wasm")]
+const fn certificate_chain_verification_cache_duration_in_seconds_default() -> u32 {
+    604800
```

Comment:
We may make more explicit that it is 1 week.

Reply:
Renamed it to `one_week_in_seconds`